### ISSUE TITLE
Reagent Bag woes. 

### ISSUE THREAD

**Issue by Oddwin** [Jaliborc/Bagnon#1546]
**Is your feature request related to a problem? Please describe.**
Getting a bit older now and my eyes get frustrated there is nothing to specifically show the reagent bag separate from the rest of the bag. 

**Describe the solution you'd like**
Either a split between regular bags and reagent bag, or possibly a color highlight?.. 

**Describe alternatives you've considered**
Alternatives? Ewww. 

**Additional context**
Bagnon is the first addon I download every single time. 

Thank you for your time.

**Comment by Dreadful-Sanity** [comment]
There is a setting under "Color Setting" for the Reagent Bank that also colours in the new reagent bags.

**Comment by yoshimo** [comment]
I prefer splitting

**Comment by Oddwin** [comment]
Color option isn't working properly for me. I'll have to do some addon surfing to find the issue. 

However.. i would prefer a split as well. and JUST do the reagent bag. I know you can split all the bags.. but that just looks messy.

**Comment by Dreadful-Sanity** [comment]
Agreed, especially since I have a weird habit of sorting my bags a certain way. I have to say I love Inventorian's tab approach.

**Comment by Woodymakespizza** [comment]
I came to post a similar request/woe. As far as I can tell, Blizz wont let us re-order/move the location of the Regeant bag, Im wondering if there would be a way for Bagnon to allow me to put the Regeant bag in another order, instead of at the bottom?

**Comment by iShark5060** [comment]
> I came to post a similar request/woe. As far as I can tell, Blizz wont let us re-order/move the location of the Regeant bag, Im wondering if there would be a way for Bagnon to allow me to put the Regeant bag in another order, instead of at the bottom?

If I remember correctly the original Blizzard bags work this way tho. If you combine all bags you still have the reagent bag extra. Currently not on a PC with WoW installed otherwise I'd check to make sure.
